app/Filament/Resources/ServerResource/Pages/EditServer.php

```diff
@@ -659,7 +664,37 @@ public function form(Form $form): Form
                                     ->deletable(false)
                                     ->addable(false)
                                     ->columnSpan(4),
-                            ])->columns(4),
+                                Forms\Components\Actions::make([
+                                    Action::make('createDatabase')
+                                        ->disabled(fn (Server $server) => DatabaseHost::query()->count() < 1 || $server->databases()->count() >= $server->database_limit)
```

Comment:
You don't need to check for the database limit, admins can usually bypass these limits.

Reply:
The create also checks limits, so it will throw an exception if we don't check the limit.

Reply:
You have to call [`setValidateDatabaseLimit`](https://github.com/pelican-dev/panel/blob/main/app/Services/Databases/DatabaseManagementService.php#L49-L58) to disable the limit check.